### ISSUE TITLE
Radiosonde app 

### ISSUE THREAD

**Issue by DavidIgor** [DarkFlippers/unleashed-firmware#717]
as Flipper is capable of receiving signal , it would be great to have ability to receive data from radiosonde (weather balloons) ...  existing weatherstation  app could be duplicated and changed  as it is fairly good interface.

**Comment by daskalovst** [comment]
This feature is only usable during the final stage of recovery when you're close to the sonde. Typically, the signal from sondes is very weak, making it nearly impossible to read the data when the balloon is far from your location. However, it's useful because it eliminates the need for additional hardware. The flipper's pocket size makes it perfect for use in dense forests, especially when you already have a rough idea of the balloon's location. It can save a lot of time walking around the forest area.

**Comment by DavidIgor** [comment]
absolutely, but with external cc1101 module and good antenna... it may be way more capable

**Comment by derskythe** [comment]
@DavidIgor 
in my region I can't find any radisonde even on HackRF with good antenna.
I mean, to create feature we need to test it IRL

**Comment by mishamyte** [comment]
Hi @DavidIgor!

It doesn't look like it's something can and will be implemented now. Closing
